dataactbroker/routes/file_routes.py

```diff
@@ -38,7 +38,8 @@ def upload_dabs_files():
     @convert_to_submission_id
     @requires_submission_perms('reader')
     @use_kwargs({'type': webargs_fields.String(missing='')})
```

Comment:
If we have things that fill in a `missing` is it still going to throw an error if we don't pass it? If so, then is there even a point in having that `missing` param since it won't REALLY fill it in to begin with?